transactionparser/README.md

```diff
@@ -80,3 +79,35 @@ The `ORDER_STATUS` is a string that represents the status of the order in the le
 *   `"cancelled"`: The transaction canceled the order. The values of `quantity` and `totalPrice` are the values of the order prior to cancellation.
 
 The `EXPIRATION_TIME` is an ISO 8601 timestamp representing the time at which the order expires (if there is an expiration time).
+
+### parseChannelChanges(metadata)
+
+Takes a PayChannel metadata object and computes the changes to the payment channel caused by the transaction. It also returns additional details about the payment channel.
+
+The return value is a JavaScript object in the following format:
+
+```javascript
+{
+    status: 'created' | 'modified' | 'deleted',
+    channelId: HEX_STRING,
```

Comment:
snakes vs camels! fight!~

Reply:
Standard for `ripple-lib` seems to be camelCase.